advanced/Scripts/update.sh

```diff
@@ -79,11 +73,15 @@ if [ ! -d "/var/www/html/admin" ]; then #This is unlikely
 fi
 
 echo "::: Checking for updates..."
-piholeVersion=$(pihole -v -p -c)
-piholeVersionLatest=$(pihole -v -p -l)
+# Checks Pi-hole version > pihole only > current local git repo version : returns string in format vX.X.X
+piholeVersion=$(/usr/local/bin/pihole -v -p -c)
```

Comment:
quotes on right of assignment as convention / glob prevention?  line 79 line 82 line 84 too ?